nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/admin/SetRequestTracingConfiguration.java

```diff
@@ -1,5 +1,4 @@
 /*
- * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
```

Comment:
I don't think I actually removed this, I just deleted the SetRequestTracingConfiguration file and renamed RequestTracingConfigurer to SetRequestTracingConfiguration. So I think git is getting this wrong 😆